packages/lib-utils/src/types/k8s.ts

```diff
@@ -50,11 +64,64 @@ export type QueryParams = Partial<{
   labelSelector: string;
   fieldSelector: string;
   resourceVersion: string;
-  [key: string]: string;
+  [key: string]: string | number;
```

Comment:
Number is required for specifiying `limit: paginationLimit,` in the queryparams. See https://github.com/openshift/dynamic-plugin-sdk/pull/37/files#diff-052548bf85cbe16601d8136f8af94f09dfa8c038d68d2affd1d9eb1a8b06e4ebR126

Reply:
Hmmm, no way to cast eh? I guess we probably want it to be a number for purity reasons 🤔